src/components/TextField/TextField.tsx

```diff
@@ -40,14 +52,15 @@ export const TextField: React.FC<Props> = ({
       <div className="control">
         <input
           type="text"
+          name={name}
           id={id}
           data-cy={`movie-${name}`}
           className={classNames('input', {
             'is-danger': hasError,
           })}
           placeholder={placeholder}
           value={value}
-          onChange={event => onChange(event.target.value)}
+          onChange={event => onChange(event)}
```

Comment:
```suggestion
          onChange={onChange}
```
additional function wrapper is redundant